tabcmd/commands/user/user_data.py

```diff
@@ -38,31 +81,33 @@ class UserCommand(Server):
 
     # used in createusers, createsiteusers
     @staticmethod
-    def validate_file_for_import(csv_file, logger, detailed=False):
+    def validate_file_for_import(csv_file: io.TextIOWrapper, logger, detailed=False, strict=False) -> int:
         num_errors = 0
         num_lines = 0
```

Comment:
What does num_lines mean? It is not clear what we are returning here.

Reply:
I added a method comment to explain it - num_lines is counting the number of lines we read in the file altogether. It could probably be removed.